crates/adapters/okx/src/http/client.rs

```diff
@@ -34,7 +34,7 @@
 //! | Funding & asset balances             | <https://www.okx.com/docs-v5/en/#rest-api-funding>                       |
 
 use std::{
-    collections::HashMap,
+    collections::{HashMap, VecDeque},
```

Comment:
decided to use `VecDeque` here since u also recommended to use this in indicators, @cjdsellers